### PR TITLE
New configuration for CANdy issuer

### PR DESCRIPTION
**Pull request by ianco**
Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

See https://github.com/bcgov/essential-services-delivery/issues/108

### CODE REVIEW

File: openshift/templates/issuer-web/config/openvp-candy/dev/config.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "displayOnFirstVisit": false
+  },
+  "inviteRequired": false,
+  "authentication": {
+    "enabled": false,
+    "autoSignOut": true,
+    "oidcSettings": {
+      "authority": "https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca",
+      "clientId": "openvp-dev",
+      "redirectUri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
```
Comment (esune) [inline openshift/templates/issuer-web/config/openvp-candy/dev/config.json]: URLs in configuration files need to be updated with the appropriate suffixes (in this case, `openvp-candy`).

Applies to other `config.json` files as well.

File: openshift/templates/issuer-web/config/openvp_rev/dev/config.json [hunk 2]
```diff
@@ -14,21 +14,21 @@
     "oidcSettings": {
       "authority": "https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca",
       "clientId": "openvp-dev",
-      "redirectUri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
-      "redirect_uri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback-error",
+      "redirectUri": "https://openvp-candy-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
```
Comment (esune) [inline openshift/templates/issuer-web/config/openvp_rev/dev/config.json]: This needs to be reverted, as it is for an existing profile. Applies to other files in `../config/openvp_rev` as well.

I think you meant to update ` openshift/templates/issuer-web/config/openvp-candy_rev/dev/config.json` instead.

**Review by esune — CHANGES_REQUESTED** [review]
If you do not need to issue revocable credentials, you could remove everything related to the `openvp-candy_rev` profile as well as lines 19-36 in `openshift/templates/api/config/openvp-candy/schemas.json` (if the services are not required you may as well not have to deploy and test them, but am not sure about what the requirements are).

**Comment by ianco** [comment]
> If you do not need to issue revocable credentials, you could remove everything related to the `openvp-candy_rev` profile as well as lines 19-36 in `openshift/templates/api/config/openvp-candy/schemas.json` (if the services are not required you may as well not have to deploy and test them, but am not sure about what the requirements are).

@swcurran  do we need to support revocation for the new CANdy issuer?

File: openshift/templates/issuer-web/config/openvp_rev/dev/config.json (hunk 2 above)
Comment (ianco) [reply openshift/templates/issuer-web/config/openvp_rev/dev/config.json]: Argh thanks for the keen eyes

**Comment by ianco** [comment]
@esune code is cleaned up and revocable version of schema (for CANdy network) is removed

**Review by esune — DISMISSED** [review]
Looks good to me! 👍🏻

**Comment by WadeBarnes** [comment]
@ianco, are these configurations dependent on the ones from this PR; https://github.com/bcgov/a2a-trust-over-ip-configurations/pull/76

**Comment by ianco** [comment]
> bcgov/a2a-trust-over-ip-configurations#76

This PR is for a new issuer (to issue creds against the CSNdy dev network).

The other PR is to update the verifier to multi-ledger (to verify creds against CANde dev as well as SOVRIN and BCovrin test)

So they are independent in some ways (they can be deployed independently) however we can't do an end-to-end demo on CANdy dev without both.

**Review by esune — APPROVED** [review]

**Pull request merged**